### DISCUSSION TITLE
Official Languages while presenting / Langues officielles durant une presentation

### DISCUSSION THREAD

**Discussion by JulieBB23** [gc-da11yn/gc-da11yn.github.io#594] [Best Practices (Open Discussions)]
Je souhaite lancer une discussion sur le forum pour explorer les stratégies de gestion de l'accessibilité et de l'équité entre les deux langues officielles. Je suis particulièrement intéressé par ce sujet et j'aimerais qu'ensemble nous recherchions des solutions.
Par exemple, participer à une présentation bilingue où le présentateur parle une langue et la diapositive PowerPoint était dans l’autre langue, puis l’inverse pour la diapositive suivante, cognitivement, ça devient un défi de taille.

J'aimerais savoir comment d'autres gèrent cette situation et quelles pratiques pourraient être adoptées pour améliorer l'expérience de tous.

__________________________________

I would like to initiate a discussion on the forum to explore strategies for managing accessibility and equity between the two official languages. I am particularly interested in this topic and would like us to search for solutions together.

As an example, participating in a bilingual presentation where the presenter speaks one language and the PowerPoint slide is in another language, and then the reverse for the next slide, cognitively becomes a significant challenge.

I would like to know how others manage this situation and what practices could be adopted to improve the experience for everyone.

**Comment by JulieBB23** [comment]
@k-w-1  and @Syracopoulos  maybe you have idea or best practices to share?

**Comment by shawnthompson** [comment]
Personally I find it very hard even if I am bilingual during bilingual meetings. I would like to see us using more translation tools that are available to use and only have meetings in one language. 

Another cognitive load is when a presenter if trying to speak in a second, sometimes third, language but is struggling to do so and we can't understand them. I appreciate the language policies but we are in 2024 now and can we not use technology that exists?

We've been expecting people who need captions to use live captions even if it's not 100% but live translations are not good enough? I find that a little ableist.

Right now I'm just talking about meetings where there are presentations, maybe once we get that right, move on to group meetings where we can all speak the official language of our choice. And then after that, any language we are comfortable with. Now that would be accessibility for **all**!

**Comment by martynka** [comment]
For me a **user with cognitive, language and learning disabilities** bilingual events are very difficult. I don't get much from those meetings.  I find hard time concentrating with the constant switching from one language to another. The information itself is overwhelming enough for my brain.

The best solution for someone like me is to have **unilingual events**. If this can't be done here are some options.

- **Prerecord presentations**, which will allow for smooth transitions and profred captions and transcripts. The automatic translations are not good. If this can't be done. 
- Provide interpreters and translation services:
   - **Have a good quality interpreters** to translate spoken language in real-time.
       -  My experience varies. Some interpreters have heavy accents and are hard to understand.  
- Provide real time captioning (**CART services**) in both languages. 
- Provide dual language presentations side by side in real settings. For online meetings, **share presentations live** and have one slide in English and one Slide in French.
- Appoint a **bilingual moderator** not only to manage the discussions to ensure that everyone has a chance to speak but also to signal when a language switch is happening to help with navigating the slides, or when additional explanations are needed.
